lineapy/annotations/external/dask.annotations.yaml

```diff
@@ -0,0 +1,64 @@
+- module: dask.dataframe
+  annotations:
+    - criteria:
+        function_names:
+          - read_sql
+          - read_sql_query
+          - read_sql_table
+      side_effects:
+        - dependency:
+            external_state: db
+    - criteria:
+        function_names:
+          - read_csv
+          - read_table
+          - read_fwf
+          - read_parquet
+          - read_hdf
+          - read_json
+          - read_orc
+      side_effects:
+        - dependency:
+            external_state: file_system
+- module: dask.dataframe
+  annotations:
+    - criteria:
+        class_method_names:
+          - to_sql
+        class_instance: DataFrame
+      side_effects:
+        - mutated_value:
+            external_state: db
+    - criteria:
+        class_method_names:
+          - to_csv
+          - to_json
+          - to_html
+          - to_hdf
+          - to_feather
```

Comment:
i also did not find any doc for to_feather. is this new? the best if found was this issue: https://github.com/dask/dask/issues/1277 that says they wont support it.